client/post-editor/verify-email-dialog.jsx

```diff
@@ -43,6 +43,16 @@ class VerifyEmailDialog extends React.Component {
 		}.bind( this ) );
 	}
 
+	getResendbuttonLabel() {
```

Comment:
From my understanding, "resend" and "button" are two separate words. If that's the case, we should put it as `getResnedButtonLabel` for consistency.